docs/landing/static/s/lib/delighted.js

```diff
@@ -0,0 +1,29 @@
+export function initialize() {
```

Comment:
This looks off - as this only exports functions and I can't see how anything is actually loaded. My js is rusty so I may be wrong. However, it looks to me that `delighted_snippet.js` has the loading mechanism and calls the survey via `delighted.survey();`.  Where as the `docs_delighted_implementation.js` has the configuration mechanism. 

If you add both to this js file does testing with `?delighted=test` still work?

Reply:
I have contacted Gilberto to help with this.